app/helpers/document_helper.rb

```diff
@@ -13,6 +13,14 @@ def evidence_tag?(tag)
     Document::EVIDENCE_TAGS.include?(tag)
   end
 
+  def supporting_document_tag?(tag)
+    Document::SUPPORT_TAGS.include?(tag)
+  end
+
+  def irrelevant_tags(planning_application, key)
+    Document.send("#{key}_tags") - planning_application.application_type.document_tags[key]
```

Comment:
Lets try to avoid `send` - at a minimum it should be `public_send` and preferably not used at all. Not that performance is an issue with this call site but it prevents the JIT compiler from getting any performance benefit and bypasses the Ruby method cache.